po/de.po

```diff
@@ -29,34 +29,34 @@ msgstr ""
 msgid " (shared with the OS)"
 msgstr " (gemeinsam mit dem Betriebssystem freigegeben)"
 
+#: pkg/kubernetes/views/node-delete.html:8
+msgid " 1\"Do you want to delete the following Nodes?"
+msgstr "1\"Sind Sie sicher, dass Sie die folgenden Nodes löschen möchten?"
```

Comment:
Sorry, kubernetes and ovirt got removed from the upstream code base. I'll send a PR to refresh translations, so that they get cleaned up. Zanata does that automatically, as the PO template is up to date. As such I'd *really* suggest to update things right here: https://fedora.zanata.org/iteration/view/cockpit/master.